webapp/app/elster_client/elster_client.py

```diff
@@ -230,14 +231,33 @@ def _generate_est_request_data(form_data, year=VERANLAGUNGSJAHR):
         # no user should come until that point without an unlock code, but they should certainly not be able to send a tax declaration
         logger.warning('Elster_Client: User without unlock code tried to send tax declaration.')
         raise TaxDeclarationNotDigitallySigned
-    
+
     meta_data = {
         'year': year,
     }
 
     return {'est_data': adapted_form_data, 'meta_data': meta_data}
 
 
+def _set_names_for_merkzeichen(adapted_form_data):
```

Comment:
Could you add a comment that we have to delete this once all fields have been renamed?